src/components/forms/ProjectGrantsForm.tsx

```diff
@@ -590,7 +589,8 @@ export const ProjectGrantsForm: FC = () => {
             </FormLabel>
 
             <PageText as='small' fontSize='helpText' color='brand.helpText'>
-              For example, do you have an MVP? Please describe its functionality and limitations and provide a link to it here.
+              For example, do you have an MVP? Please describe its functionality and limitations and
+              provide a link to it here.
```

Comment:
seems prettier config is formatting these?

Reply:
yea... 🤔 not sure why, I guess that for some reason my prettier didn't detect the project's config. Will double-check it soon and fix these modifications

Reply:
This is actually correct. We have a `"printWidth": 100`. Seems that were not formatted correctly before.

Reply:
Nice.